servicecontrol/transports.md

```diff
@@ -66,4 +66,8 @@ The following ServiceControl connection string options are available in versions
 * `QueueNamePrefix=<value>` - Queue name prefix transport [option](/transports/sqs/configuration-options.md#queuenameprefix),
 * `TopicNamePrefix=<value>` - Topic name prefix transport [option](/transports/sqs/configuration-options.md#topicnameprefix)
 * `S3BucketForLargeMessages=<value>` - S3 bucket for large messages [option](/transports/sqs/configuration-options.md#s3bucketforlargemessages),
-* `S3KeyPrefix=<value>` - S3 key prefic [option](/transports/sqs/configuration-options.md#s3bucketforlargemessages-s3keyprefix).
+* `S3KeyPrefix=<value>` - S3 key prefix [option](/transports/sqs/configuration-options.md#s3bucketforlargemessages-s3keyprefix).
+
+The following ServiceControl connection string options are available in versions 4.29.3 and above:
```

Comment:
I wonder if we should do this. It seems historically we haven't really documented from what version on a certain setting was introduced. Given that SC has a short lifespan this information is probably anyway not that important. I vote to just mention the setting

Reply:
I have removed both notes about versions from that section.